README.md

```diff
@@ -251,9 +251,98 @@ Congratulations! You can now send remote push notifications to your device from
 		```
 		
 3. ##### InApp Notifications
-	To display the user's InApp notifications call `spawnInAppNotification` with a defined `ITEActionBlock` callback handler. When a user clicks a button on the notification, the defined handler is called and passed the action name defined in the InApp template.
+	###### Default Behavior
+	By default, when an inApp message arrives from the server it is automatically shown by the SDK provided the app is in foreground. If an inApp message is already showing when the message arrives, the new inApp will be shown 30 seconds (see how to change this default value below) after the currently displaying inApp is closed. Once an inApp message is shown, it will be "consumed" from the server queue and removed from the local queue as well. There is no need to write any code to get this default behavior. 
 	
-	InApp opens and button clicks are automatically tracked when the notification is called via `spawnInAppNotification`. Using `spawnInAppNotification`, the notification is consumed and removed from the user's in-app messages queue. If you want to retain the messages on the queue, look at using `getInAppMessages` directly. If you use `getInAppMessages` you will need to manage the in-app opens manually in the callback handler.		
+	###### Overriding Whether to Show or Skip a Particular InApp Message
```

Comment:
- No need to indent
- Capitalization: Overriding whether to show or skip a particular in-app message